video_xblock/static/html/fields/boolean.html

```diff
@@ -3,9 +3,9 @@
   id="xb-field-edit-{{field.name}}"
 >
   <option value="1" {% if field.value %}selected{% endif %}>
-    True {% if field.default %}&nbsp;&nbsp;&nbsp;&nbsp;(Default){% endif %}
+    True {% if field.default %}&emsp;&emsp;&emsp;&emsp;(Default){% endif %}
```

Comment:
He actually suggested using one `&emsp;` instead of 4 `&nbsp;`, because former one is wider.
See here -- https://www.cs.tut.fi/~jkorpela/chars/spaces.html